symplyphysics/laws/hydro/volume_flux_is_constant.py

```diff
@@ -0,0 +1,40 @@
+from sympy import Eq, solve
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The product of the area and the fluid speed, which is called volume flux, is constant 
+## at all points along the tube of flow of an incompressible liquid.
+
+# Law: A*v = constant
```

Comment:
We usually write constant bound laws as derivative equals zero, eg https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/conservation/mechanical_energy_is_constant.py
Maybe write an additional form of this law? Is there a derivative version of this law for reference?

Reply:
Also instead of area_start, area_end we can use a bit more pleasant area(time_before), like here: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/conservation/momentum_after_collision_equals_to_momentum_before.py

Reply:
Okay, I took inspiration from https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/thermodynamics/volume_is_constant.py and the like.

Reply:
I see. The preferred way depends, on whether area and velocity are non-linear functions. If they usually are, it is better to have a derivative.